src/components/CompareResults/Distribution.tsx

```diff
@@ -33,6 +35,7 @@ function Distribution(props: DistributionProps) {
     name: 'Base',
     median: baseMedian,
     values: baseRuns,
+    replicates: baseRunsReplicates,
```

Comment:
I'd do this instead:
```suggestion
    values: baseRunsReplicates.length ? baseRunsReplicates : baseRuns,
```

And remove all the other changes.

I understand you wanted to keep both values but it doesn't seem useful in this component.

Reply:
(same for the "new" of course)